modules/savedb.py

```diff
@@ -0,0 +1,62 @@
+import time
+import json
+import MySQLdb
+import sys
+import argparse
+
+def saveToDatabase(database, user, password, links):
+        """
+        Connects to a MYSQL DB
+        Create MYSQL Table
+        Add link to the DB
+
+        Args:
+                data = data that is being stored in the database
+                user = username to login into MYSQL
+                password = password of MYSQL
+                link = URLs from the crawler
+        """
+        if database and user and password is None:
```

Comment:
Don't you think (True is None or False is None) going to evaluate to False and hence in else loop???

Reply:
It is just to prevent null values and exits with a message... This can be improved

Reply:
Could you switch it to `if not database and not user and not password`?

Reply:
yes, ofc... report here all the necessary refactoring and in aprox. a week I will commit all the changes